### PR TITLE
Fixed bug lol

### PR DESCRIPTION
**Pull request by macaquedev**
@ykr5184 your Scoop part 1 solution had a tiny bit of missing latex and I've added it 

image

### CODE REVIEW

File: solutions/bio1/Juggluging/index.tsx
Comment (myst-6) [inline solutions/bio1/Juggluging/index.tsx:1]: i don't think this file is meant to be on this PR.. not only this but it contains the old jambloat!

**Review by myst-6 — CHANGES_REQUESTED** [review]
oops

File: solutions/bio1/Juggluging/index.tsx
Comment (macaquedev) [reply solutions/bio1/Juggluging/index.tsx:1]: oh dear. I clearly pushed the wrong branch.

Comment (macaquedev) [reply solutions/bio1/Juggluging/index.tsx:1]: I'm going to open a new PR.

Comment (macaquedev) [reply solutions/bio1/Juggluging/index.tsx:1]: please disregard this one.

**Pull request closed without merging**